src/main/kotlin/day15/Repository/Comments.Repository.kt

```
@@ -0,0 +1,31 @@
+    override var comments: List<Comment>
+        get() {
+            return parseJson(jsonDataSource)
+        }
+        set(value) {}
+
+    override suspend fun getComments(postId: Int): List<Comment> {
+        try {
+            return comments.filter { comment ->
+                comment.postId == postId
+            }
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)